### PR TITLE
Add two new Zerto Templates

### PR DESCRIPTION
**Pull request by shannonsnowden**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by navalev** [comment]
@bmoore-msft - seems that the validation process fails due to Marketplace purchase eligibility of the tested subscription. The templates in this PR deploy a Zerto VM marketplace image (programmatic deployment should be enabled on the subscription). How can we resolve this?

File: 201-zerto-zca-vpn/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Deploy Zerto Virtual Replication to Azure with a VPN
+
+This template deploys the Zerto Cloud Appliance (ZCA) to Azure. It deploys a virtual network, public ip, network interface and a ZCA virtual machine. 
+
+It also configures an [Azure VPN Gateway](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-howto-site-to-site-resource-manager-portal) to create a site-to-site VPN connection between the Azure Virtual Network and your local on-premises network.
+
+
+1. [azuredeploy.json](./azuredeploy.json)
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/README.md]: not sure what this is for, can we just remove it?

File: 201-zerto-zca-vpn/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FZerto-TA-Public%2Fazure-quickstart-templates%2Fmaster%2F201-zerto-zca-vpn%2Fazuredeploy.json" target="_blank">
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/README.md]: needs to be updated for Azure/master

File: 201-zerto-zca-vpn/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+         ]
+      },
+      "localGatewayName":{  
+         "type":"string",
+         "defaultValue":"localGateway",
+         "metadata":{  
+            "description":"Name for gateway resource representing your local/on-prem gateway"
+         }
+      },
+      "localGatewayIpAddress":{
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/azuredeploy.json]: what if I don't have one of these?  will the deployment fail?

File: 201-zerto-zca-vpn/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+         }
+      },
+      "virtualNetworkName":{  
+         "type":"string",
+         "defaultValue":"azureVnet",
+         "metadata":{  
+            "description":"Name for the Azure Virtual Network"
+         }
+      },
+      "newStorageAccountName":{
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/azuredeploy.json]: just move this to a variable - or use managed disks

File: 201-zerto-zca-vpn/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+            "description":"Name for the new connection between Azure VNet and other network"
+         }
+      },
+      "sharedKey":{  
+         "type":"string",
+         "metadata":{  
+            "description":"Shared key (PSK) for IPSec tunnel"
+         }
+      },
+      "vmName":{
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/azuredeploy.json]: move to variable

File: 201-zerto-zca-vpn/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,398 @@
...
+            "description":"Username for the VM"
+         }
+      },
+      "adminPassword":{  
+         "type":"securestring",
+         "metadata":{  
+            "description":"User password for the VM"
+         }
+      },
+      "storageAccountType":{
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/azuredeploy.json]: can we just set this for the user?

File: 201-zerto-zca-vpn/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,398 @@
...
+            "Premium_LRS"
+         ],
+         "metadata":{  
+            "description":"Storage Account Type"
+         }
+      }
+   },
+   "variables":{  
+      "vnetID":"[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetworkName'))]",
+      "gatewaySubnetRef":"[concat(variables('vnetID'),'/subnets/','GatewaySubnet')]",
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/azuredeploy.json]: just use resourceID and remove the vnetID above

File: 201-zerto-zca-vpn/azuredeploy.parameters.json [hunk 8]
```diff
@@ -0,0 +1,27 @@
+{  
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{      
+     "localGatewayIpAddress":{
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/azuredeploy.parameters.json]: put these in as defaultValues in the template where possible - i.e. anything not GEN'd

File: 201-zerto-zca-vpn/azuredeploy.parameters.json [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+     "newStorageAccountName":{  
+        "value": "GEN-UNIQUE"
+     },     
+     "subnetPrefix":{  
+        "value": "10.3.0.0/24"
+     },         
+     "sharedKey":{  
+        "value": "GEN-UNIQUE"
+     },     
+     "adminUsername":{
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/azuredeploy.parameters.json]: GEN-UNIQUE

File: 201-zerto-zca/README.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
+# Deploy Zerto Virtual Replication to Azure
+
+This template deploys a Zerto Cloud Appliance (ZCA) to Azure:
+
+1. [azuredeploy.json](./azuredeploy.json)
```
Comment (bmoore-msft) [inline 201-zerto-zca/README.md]: remove this?

File: 201-zerto-zca/README.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+
+This templates deploys a virtual network, public ip, network interface and a Zerto virtual machine. It also provisions a network security group to enable RDP to the VM.
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FZerto-TA-Public%2Fazure-quickstart-templates%2Fmaster%2F201-zerto-zca%2Fazuredeploy.json" target="_blank">
```
Comment (bmoore-msft) [inline 201-zerto-zca/README.md]: update for Azure/master

File: 201-zerto-zca/azuredeploy.json [hunk 12]
```diff
@@ -0,0 +1,246 @@
...
+          "Standard_ZRS",
+          "Standard_GRS",
+          "Standard_RAGRS",
+          "Premium_LRS"
+        ],
+        "metadata":{  
+            "description":"Storage Account type"
+         }
+      },
+      "vmName": {
```
Comment (bmoore-msft) [inline 201-zerto-zca/azuredeploy.json]: move to variable

File: 201-zerto-zca/azuredeploy.json [hunk 13]
```diff
@@ -0,0 +1,246 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+    "newStorageAccountName" :{
```
Comment (bmoore-msft) [inline 201-zerto-zca/azuredeploy.json]: move the storageAccount settings to a variable or just use managed disks

File: 201-zerto-zca/azuredeploy.json [hunk 14]
```diff
@@ -0,0 +1,246 @@
...
+      "vnetSubnet1Name": "Subnet-1",
+      "vnetSubnet1Prefix": "10.0.0.0/24",
+      "vnetSubnet2Name": "Subnet-2",
+      "vnetSubnet2Prefix": "10.0.1.0/24",
+      "vmImagePublisher": "MicrosoftWindowsServer",
+      "vmImageOffer": "WindowsServer",
+      "vmOSDiskName": "vmOSDisk",
+      "vmVmSize": "Standard_DS2_v2",
+      "vmVnetID": "[resourceId('Microsoft.Network/virtualNetworks', 'vnet')]",
+      "vmSubnetRef": "[concat(variables('vmVnetID'), '/subnets/', variables('vnetSubnet1Name'))]",
```
Comment (bmoore-msft) [inline 201-zerto-zca/azuredeploy.json]: just use resourceId and remove vmvnetid

File: 201-zerto-zca/azuredeploy.parameters.json [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+         "value":"zertozca"
+      },
+      "storageAccountType" : {
+          "value": "Standard_LRS"
+      },
+      "vmName" : {
+          "value": "zca-vm"
+      },
+      "vmAdminUserName":{  
+         "value":"zerto"
```
Comment (bmoore-msft) [inline 201-zerto-zca/azuredeploy.parameters.json]: GEN-UNIQUE

File: 201-zerto-zca-vpn/README.md (hunk 1 above)
Comment (navalev) [reply 201-zerto-zca-vpn/README.md]: done

**Comment by navalev** [comment]
@bmoore-msft fixed all comments

File: 201-zerto-zca-vpn/README.md [hunk 16]
```diff
@@ -1,13 +1,9 @@
 # Deploy Zerto Virtual Replication to Azure with a VPN
 
-This template deploys the Zerto Cloud Appliance (ZCA) to Azure. It deploys a virtual network, public ip, network interface and a ZCA virtual machine. 
+This templates deploys a virtual network, public ip, network interface and a Zerto Cloud Appliance (ZCA) virtual machine.
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/README.md]: typo in "templates" s/b "template"

File: 201-zerto-zca-vpn/azuredeploy.parameters.json [hunk 17]
```diff
@@ -7,21 +7,24 @@
      },
      "localAddressPrefix":{
```
Comment (bmoore-msft) [inline 201-zerto-zca-vpn/azuredeploy.parameters.json]: for parameters that don't use GEN values, move them to defaultValues in the template

File: 201-zerto-zca/README.md [hunk 18]
```diff
@@ -1,11 +1,8 @@
 # Deploy Zerto Virtual Replication to Azure
 
-This template deploys a Zerto Cloud Appliance (ZCA) to Azure:
 
-1. [azuredeploy.json](./azuredeploy.json)
+This templates deploys a virtual network, public ip, network interface and a Zerto Cloud Appliance (ZCA) virtual machine. It also provisions a network security group to enable RDP to the VM.
```
Comment (bmoore-msft) [inline 201-zerto-zca/README.md]: typo "templates" s/b "template"

**Comment by bmoore-msft** [comment]
A couple new comments and some of the old ones (subnetId) were not addressed...

File: 201-zerto-zca-vpn/azuredeploy.json (hunk 6 above)
Comment (navalev) [reply 201-zerto-zca-vpn/azuredeploy.json]: There's a default value - Standard_LRS. shouldn't we allow the user to change it?

File: 201-zerto-zca-vpn/azuredeploy.parameters.json (hunk 8 above)
Comment (navalev) [reply 201-zerto-zca-vpn/azuredeploy.parameters.json]: done

File: 201-zerto-zca/azuredeploy.json (hunk 14 above)
Comment (navalev) [reply 201-zerto-zca/azuredeploy.json]: done

File: 201-zerto-zca-vpn/azuredeploy.parameters.json (hunk 17 above)
Comment (navalev) [reply 201-zerto-zca-vpn/azuredeploy.parameters.json]: done

File: 201-zerto-zca/README.md (hunk 18 above)
Comment (navalev) [reply 201-zerto-zca/README.md]: done

File: 201-zerto-zca-vpn/README.md (hunk 16 above)
Comment (navalev) [reply 201-zerto-zca-vpn/README.md]: done

**Comment by navalev** [comment]
@bmoore-msft fixed all comments, build passed

File: 201-zerto-zca-vpn/azuredeploy.json (hunk 3 above)
Comment (navalev) [reply 201-zerto-zca-vpn/azuredeploy.json]: all have default values now

File: 201-zerto-zca-vpn/azuredeploy.json (hunk 7 above)
Comment (navalev) [reply 201-zerto-zca-vpn/azuredeploy.json]: done

**Pull request merged**